package.json

```diff
@@ -12,12 +12,16 @@
     "readline": "^1.3.0"
   },
   "devDependencies": {
+    "coveralls": "^2.13.1",
     "eslint": "^3.19.0",
+    "istanbul": "^0.4.5",
```

Comment:
I think it is, Instanbul is the tool which is actually generating the coverage data.